snp_motion_planning/src/planning_server.cpp

```diff
@@ -182,6 +185,25 @@ class PlanningServer
     : node_(node), env_(std::make_shared<tesseract_environment::Environment>())
   {
     // Declare ROS parameters
+#if __has_include(<rclcpp/version.h>) && RCLCPP_VERSION_MAJOR >= 16  // ROS 2 Humble
+    node_->declare_parameter<std::string>("robot_description");
+    node_->declare_parameter<std::string>("robot_description_semantic");
+    node_->declare_parameter<bool>(VERBOSE_PARAM, false);
+    node_->declare_parameter<std::vector<std::string>>(TOUCH_LINKS_PARAM, {});
+    node_->declare_parameter<double>(MAX_TRANS_VEL_PARAM);
+    node_->declare_parameter<double>(MAX_ROT_VEL_PARAM);
+    node_->declare_parameter<double>(MAX_TRANS_ACC_PARAM);
+    node_->declare_parameter<double>(MAX_ROT_ACC_PARAM);
+    node_->declare_parameter<bool>(CHECK_JOINT_ACC_PARAM, false);
+    node_->declare_parameter<double>(VEL_SCALE_PARAM, 1.0);
+    node_->declare_parameter<double>(ACC_SCALE_PARAM, 1.0);
+    node_->declare_parameter<double>(LVS_PARAM, 0.05);
+    node_->declare_parameter<double>(CONTACT_DIST_PARAM, 0.0);
+    node_->declare_parameter<std::string>(TASK_COMPOSER_CONFIG_FILE_PARAM);
+    node_->declare_parameter<std::string>(TASK_NAME_PARAM);
+    node_->declare_parameter<double>(OCTREE_RESOLUTION_PARAM, 0.010);
+    node_->declare_parameter<std::string>(COLLISION_OBJECT_TYPE_PARAM);
+#else  // ROS 2 Foxy
     node_->declare_parameter("robot_description");
```

Comment:
Let's try to get rid of the `#if` here. It looks like non-templated declares need to add a second argument `rclcpp::ParameterValue()`, which used to be default in foxy but is not provided in humble

Reply:
Doing this it is able to build, but gives a runtime error. Switched to just giving a default for all parameters which should be good for both foxy and humble+ versions